yarn-project/noir-contracts/src/contracts/zk_token_contract/src/main.nr

```diff
@@ -32,16 +29,16 @@ contract ZkToken {
         inputs: PrivateContextInputs,
         //*********************************/
         initial_supply: Field, 
-        owner: Field,
+        owner: Field
```

Comment:
Curious if you removed this manually or you have a noir formatter set up

Reply:
This must have been a manual change.